lib/ansible/playbook/base.py

```diff
@@ -611,6 +611,7 @@ class Base(FieldAttributeBase):
     _check_mode = FieldAttribute(isa='bool', default=context.cliargs_deferred_get('check'))
     _diff = FieldAttribute(isa='bool', default=context.cliargs_deferred_get('diff'))
     _any_errors_fatal = FieldAttribute(isa='bool', default=C.ANY_ERRORS_FATAL)
+    _forks = FieldAttribute(isa='list', extend=True, prepend=True)
```

Comment:
needs to be ` static=True` as templating per host makes no sense here.

Reply:
I have marked the forks attribute as static. Interesting is that there is only one attribute so far that is defined to be static: The register attribute.